src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
I don't think we can rely on that always working.  At the very least, we have to worry about escaping, since there's no reason a Python docstring couldn't be malformed markdown.

It seems like we might need to re-implement something like doc2md, at least in a limited form.